src/components/Controls.vue

```diff
@@ -138,8 +138,40 @@
 								<label :for="user.uid"><NcAvatar :user="user.uid" :size="24" :disable-menu="true" /> {{ user.displayname }}</label>
 							</div>
 
-							<h3>{{ t('deck', 'Filter by due date') }}</h3>
+							<h3>{{ t('deck', 'Filter by completed') }}</h3>
+							<div class="filter--item">
+								<input id="filter-option-both"
+									v-model="filter.completedOrOpen"
+									type="radio"
+									class="radio"
+									:value="true"
+									@change="setFilter"
+									@click="beforeSetFilter">
+								<label for="filter-option-both">{{ t('deck', 'both') }}</label>
+							</div>
+							<div class="filter--item">
+								<input id="filter-option-open"
+									v-model="filter.open"
+									type="radio"
+									class="radio"
+									:value="true"
+									@change="setFilter"
+									@click="beforeSetFilter">
+								<label for="filter-option-open">{{ t('deck', 'open') }}</label>
+							</div>
+
+							<div class="filter--item">
+								<input id="filter-option-completed"
+									v-model="filter.completed"
+									type="radio"
+									class="radio"
+									:value="true"
+									@change="setFilter"
+									@click="beforeSetFilter">
+								<label for="filter-option-completed">{{ t('deck', 'completed') }}</label>
+							</div>
```

Comment:
Maybe you could refactor this so that the radio buttons are actually radio button group with a common name attribute, then their value could be directly assigned to one state entry `filter.completed` that could be either `both`, `open` or `completed` and can probably simplify the check logic further down quite a bit.